### PR TITLE
Extend market details E2E tests

### PR DESCRIPTION
**Pull request by EmilMalanczak**

### CODE REVIEW

File: packages/app/src/ui/molecules/cooldown-timer/CooldownTimer.tsx [hunk 1]
```diff
@@ -12,9 +12,15 @@ interface CooldownTimerProps {
...
   forceOpen?: boolean
+  'data-testid'?: string
 }
 
-export function CooldownTimer({ renewalPeriod, latestUpdateTimestamp, forceOpen }: CooldownTimerProps) {
+export function CooldownTimer({
+  renewalPeriod,
+  latestUpdateTimestamp,
+  forceOpen,
+  'data-testid': dataTestId,
```
Comment (oskarvu) [inline packages/app/src/ui/molecules/cooldown-timer/CooldownTimer.tsx:22]: You don't need to pass test id as a prop here. You can use `testIds.marketDetails.capAutomator.cooldownTimer` directly on trigger.

File: packages/app/src/pages/MarketDetails.test-e2e.ts [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+  const DAI = '0x6B175474E89094C44Da98b954EedeAC495271d0F'
+  const WETH = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+  const WEETH = '0xcd5fe23c85820f7b72d0926fc9b05b43e359b7ee'
+  const USDC = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
+  const WBTC = '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599'
+
+  const fork = setupFork({
+    blockNumber: CAP_AUTOMATOR_BLOCK_NUMBER,
+    chainId: mainnet.id,
+    simulationDateOverride: new Date(1724846808220),
```
Comment (oskarvu) [inline packages/app/src/pages/MarketDetails.test-e2e.ts:23]: Can you use human readable value instead of a number in date constructor?

**Review by oskarvu — APPROVED** [review]

**Review by oskarvu — COMMENTED** [review]
Looks good, just some tests are failing.

File: packages/app/src/pages/MarketDetails.test-e2e.ts [hunk 3]
```diff
@@ -300,4 +306,170 @@ test.describe('Market details', () => {
...
+  const WETH = '0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1'
+  const USDT = '0x4ECaBa5870353805a9F068101A40E0f32ed605C6'
+
+  const fork = setupFork({
+    blockNumber: GNOSIS_DEFAULT_BLOCK_NUMBER,
+    chainId: gnosis.id,
+    simulationDateOverride: new Date(1724846808220),
+  })
+
+  test.describe('Cap automator', () => {
```
[the remark this answers is not in the document]
Comment (krzkaczor) [reply packages/app/src/pages/MarketDetails.test-e2e.ts:412]: i think it's fine.

File: packages/app/src/pages/MarketDetails.test-e2e.ts [hunk 4]
```diff
@@ -300,4 +306,170 @@ test.describe('Market details', () => {
...
+      await expect(marketDetailsPage.locatePanelAutomatorMaxCap(supplyPanelLocator)).not.toBeVisible()
+
+      const borrowPanelLocator = marketDetailsPage.locateBorrowStatusPanel()
+      await marketDetailsPage.expectPanelCap(borrowPanelLocator, '16M WXDAI')
+      await expect(marketDetailsPage.locatePanelAutomatorMaxCap(borrowPanelLocator)).not.toBeVisible()
+
+      await expect(marketDetailsPage.locateCollateralStatusPanel()).not.toBeVisible()
+    })
+
+    test('USDT', async ({ page }) => {
```
Comment (krzkaczor) [inline packages/app/src/pages/MarketDetails.test-e2e.ts:455]: IMO we could delete this case as it's the same as WETH -- no cap automator, not a native asset, borrow and supply enabled...

**Review by krzkaczor — APPROVED** [review]

**Review by oskarvu — APPROVED** [review]

**Pull request merged**